isdproject/src/Login.js

```diff
@@ -22,8 +22,23 @@ const Login = () => {
     e.preventDefault();
     try {
       const res = await axios.post('http://localhost:8080/api/auth/login', formData);
+
       const { token } = res.data;
+      console.log(token);
       Cookies.set('token', token, { expires: 1 / 24 }); // Expires in 1 hour
+        try {
```

Comment:
Double-check this debugging code and remove

Reply:
This content was deleted